src/CMakeLists.txt

```diff
@@ -80,9 +77,20 @@ endif()
 get_target_property(ESPP_CFLAGS _espressopp COMPILE_OPTIONS)
 message(STATUS "_espressopp compiler flags are: ${ESPP_CFLAGS}")
 
-target_link_libraries(_espressopp ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} MPI::MPI_CXX ${FFTW3_LIBRARIES} ${VAMPIRTRACE_LIBRARIES})
+target_include_directories(_espressopp PUBLIC ${CMAKE_SOURCE_DIR}/src)
+target_include_directories(_espressopp PUBLIC ${CMAKE_SOURCE_DIR}/src/include)
+target_include_directories(_espressopp PUBLIC ${CMAKE_BINARY_DIR}/src)
+target_include_directories(_espressopp PRIVATE ${FFTW3_INCLUDES})
```

Comment:
We should create an imported target in our custom `FindFFTW3.cmake`.

See  https://github.com/votca/tools/blob/master/CMakeModules/FindFFTW3.cmake#L40-L51

Reply:
Also true for `mpi4py`. Anyone volunteering? :)

Reply:
I would leave that until #351 is done.